src/Halogen/Internal/VirtualDOM.purs

```diff
@@ -128,7 +128,19 @@ foreign import vnode
   \  return function(props) {\
   \    return function(children) {\
   \      var VirtualNode = require('virtual-dom/vnode/vnode');\
-  \      return new VirtualNode(name, props, children);\
+  \      var props1 = {\
+  \        attributes: {}\
+  \      };\
+  \      for (var key in props) {\
```

Comment:
That would need a polyfill for ES3 right? Which is fine obviously, but would need documenting.